tests/ResponseTest.php

```diff
@@ -126,4 +185,97 @@ public function testIsType()
             $this->assertSame($digit == 5, $response->isServerError());
         }
     }
+
+    public function testHasHeader()
+    {
+        $response = new Response();
+        //$resultTrue = $response->
+
+        $this->assertFalse($response->hasHeader('no_header_name'));
+    }
+
+    public function testAddHeaderWithArrayValue()
+    {
+        $response = new Response();
+
+        $this->assertInstanceOf(Response::class, $response->addHeader('Date', ['Sun', '18 Oct 2009 08:56:53 GMT']));
+        $this->assertSame('Sun, 18 Oct 2009 08:56:53 GMT', $response->getHeader('Date'));
+    }
+
+    public function testAddHeaderWithArrayValueAppendExistedHeader()
+    {
+        $response = new Response();
+        $response->addHeader('Date', 'Sun');
+        $response->addHeader('Date', ['18 Oct 2009 08:56:53 GMT']);
+
+        $this->assertSame('Sun, 18 Oct 2009 08:56:53 GMT', $response->getHeader('Date'));
+    }
+
+    public function testAddHeaders()
+    {
+        $response = new Response();
+        $result = $response->addHeaders([
+            'Content-Type' => 'text/html; charset=utf-8',
+            'Server' => 'Apache',
+        ]);
+
+        $this->assertInstanceOf(Response::class, $result);
+        $this->assertSame('text/html; charset=utf-8', $response->getHeader('Content-Type'));
+        $this->assertSame('Apache', $response->getHeader('Server'));
+    }
+
+    public function testGetHead()
+    {
+        $response = new Response();
+        $response->addHeaders([
+            'Content-Type' => 'text/html; charset=utf-8',
+            'Server' => 'Apache',
+        ]);
+        $expectedHead = 'HTTP/1.0 200 OK' . "\r\n" . 'Content-Type: text/html; charset=utf-8' . "\r\n" . 'Server: Apache' . "\r\n\r\n";
+
+        $this->assertSame($expectedHead, $response->getHead());
+    }
+
+    public function testGetContentLength()
+    {
+        $response = new Response();
+
+        $this->assertSame(0, $response->getContentLength());
+    }
+
+    public function testGetContentLengthShouldReturn19730()
+    {
+        $response = new Response();
+        $response->addHeader('Content-Length', 19730);
+
+        $this->assertSame(19730, $response->getContentLength());
+    }
+
+    public function testIsContentType()
+    {
+        $response = new Response();
+        $response->addHeader('Content-Type', 'text/html; charset=utf-8');
+
+        $this->assertTrue($response->isContentType('text/html'));
```

Comment:
It would be good to test a few cases here (test that it can return false as well), and maybe with/without the `;` part.  That's at least 4 tests, which makes it a good fit for a data provider.